README.rst

```diff
@@ -0,0 +1,47 @@
+|Icon| |title|_
+===============
+
+.. |title| replace:: bg-cookiecutter
+.. _title: https://Billingegroup.github.io/cookiecutter
+
+.. |Icon| image:: https://avatars.githubusercontent.com/Billingegroup
+        :target: https://Billingegroup.github.io/cookiecutter
+        :height: 100px
+
+|PyPi| |Forge| |PythonVersion| |PR|
+
+|CI| |Codecov| |Black| |Tracking|
+
+.. |Black| image:: https://img.shields.io/badge/code_style-black-black
+        :target: https://github.com/psf/black
+
+.. |CI| image:: https://github.com/Billingegroup/cookiecutter/actions/workflows/matrix-and-codecov-on-merge-to-main.yml/badge.svg
+        :target: https://github.com/Billingegroup/cookiecutter/actions/workflows/matrix-and-codecov-on-merge-to-main.yml
+
+.. |Codecov| image:: https://codecov.io/gh/Billingegroup/cookiecutter/branch/main/graph/badge.svg
+        :target: https://codecov.io/gh/Billingegroup/cookiecutter
+
+.. |Forge| image:: https://img.shields.io/conda/vn/conda-forge/bg-cookiecutter
+        :target: https://anaconda.org/conda-forge/bg-cookiecutter
+
+.. |PR| image:: https://img.shields.io/badge/PR-Welcome-29ab47ff
+
+.. |PyPi| image:: https://img.shields.io/pypi/v/bg-cookiecutter
+        :target: https://pypi.org/project/bg-cookiecutter/
+
+.. |PythonVersion| image:: https://img.shields.io/pypi/pyversions/bg-cookiecutter
+        :target: https://pypi.org/project/bg-cookiecutter/
+
+.. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
+        :target: https://github.com/Billingegroup/cookiecutter/issues
+
+``bg-cookiecutter`` is a Python package standard and generator for scientific code. Use ``bg-cookiecutter`` to launch a new project or migrate existing ones to support the latest Python versions and streamline the process of distributing and maintaining your software package.
```

Comment:
Since we already have a thorough documentation via our official doc (containing contribution guide, FAQs, etc), I was thinking maybe just have a sales pitch and have a getting started section so that we don't have to maintain both readme.rst as well as the doc?

Reply:
we do want to have the "if you use this, cite our paper" section, wherever that goes.

Reply:
done. added to readme.rst - will add this again in our doc as well - separate PRs